bedrock/exp/templates/exp/firefox/index.de.html

```diff
@@ -0,0 +1,250 @@
+              <li class="mzp-c-menu-list-item"><a href="/firefox/download/thanks/" data-link-type="Desktop" data-download-os="Desktop" data-download-version="standard">Desktop</a></li>
+            <!--<![endif]-->
+            <li class="mzp-c-menu-list-item"><a href="{{ firefox_adjust_url('android', 'firefox_home') }}" rel="external noopener" data-link-type="download" data-display-name="Android" data-download-version="android" data-download-os="Android"> Android</a></li>
+            <li class="mzp-c-menu-list-item"><a href="{{ firefox_adjust_url('ios', 'firefox_home') }}" rel="external noopener" data-link-type="download" data-display-name="iOS" data-download-version="ios" data-download-os="iOS">iOS</a></li>
+          </ul>
+        </div>
+        <p class="js-fx-only"><a class="mzp-c-cta-link" href="https://addons.mozilla.org/firefox/addon/facebook-container/{{ referrals }}">Hol dir die Facebook-Container-Erweiterung</a></p>
+      {% endcall %}
+    </div>
+
+    <div class="mzp-l-content">
+        {% call feature_card(
+          title='Firefox Monitor',
+          heading_level=3,
+          image_url='img/firefox/home/master/monitor.svg',
+          class='mzp-l-card-feature-right-half t-monitor',
+          link_url='https://monitor.firefox.com/' + referrals,
+          link_cta='Melde dich für Datenleak-Berichte an',
+          ga_title='Monitor',
+          media_after=True
+        ) %}
+        <h4>Sei Hackern einen Schritt voraus und bleib informiert</h4>
+        {% endcall %}
+    </div>
+
+    <div class="mzp-l-content">
+      {% call feature_card(
+        title='Firefox Lockwise',
+        heading_level=3,
+        image_url='img/firefox/home/master/lockwise.svg',
+        class='mzp-l-card-feature-left-half t-lockwise',
+        ga_title='Lockwise',
+        media_after=True
+      ) %}
+      <h4>Bewahre deine Passwörter sicher auf – auf allen Geräten</h4>
+        <div class="mzp-c-menu-list mzp-t-cta mzp-t-download">
+          <h5 class="mzp-c-menu-list-title">App downloaden</h5>
+          <ul class="mzp-c-menu-list-list" id="menu-lockwise">
+            <li class="mzp-c-menu-list-item"><a href="{{ lockwise_adjust_url('android', 'firefox-home') }}" rel="external noopener" data-cta-type="link" data-cta-text="Download Lockwise for Android">Android</a></li>
+            <li class="mzp-c-menu-list-item"><a href="{{ lockwise_adjust_url('ios', 'firefox-home') }}" rel="external noopener" data-cta-type="link" data-cta-text="Download Lockwise for iOS">iOS</a></li>
+          </ul>
+        </div>
+        <p><a class="mzp-c-cta-link" href="{{ url('firefox.lockwise.lockwise') }}" rel="external noopener" data-cta-type="link" data-cta-text="Lockwise Learn More">Weitere Infos zu Lockwise</a></p>
+      {% endcall %}
+    </div>
+
+    <div class="t-promise mzp-t-dark">
+      <div class="mzp-l-content">
+          {% call feature_card(
+            title='Bekomme den <strong>Respekt</strong>, den du verdienst',
```

Comment:
Think you need to avoid auto escaping the `<strong>` here

```suggestion
            title='Bekomme den <strong>Respekt</strong>, den du verdienst'|safe,
```